packs/libcloud/actions/create_vm.py

```diff
@@ -32,11 +31,10 @@ def run(self, credentials, name, size_id=None, image_id=None, size_name=None, im
             size = [s for s in driver.list_sizes() if size_name in s.name][0]
 
         if image_id is not None:
-            image = NodeImage(id=image_id, name=None,
-                              driver=driver)
+            image = [i for i in driver.list_images() if image_id == i.id][0]
```

Comment:
In theory, existing code should work since the drivers should really only just need an `id` for create_node.

It seems like though that some driver is using some other attribute (name?) and not id.